packages/fluentui/react-northstar/src/components/TextArea/TextArea.tsx

```diff
@@ -43,65 +52,97 @@ export interface TextAreaProps extends UIComponentProps, ChildrenComponentProps
   fluid?: boolean;
 }
 
-export interface TextAreaState {
-  value?: TextAreaProps['value'];
-}
+export type TextAreStylesProps = Required<Pick<TextAreaProps, 'inverted' | 'resize' | 'fluid' | 'disabled'>>;
 
 export const textAreaClassName = 'ui-textarea';
 
-class TextArea extends AutoControlledComponent<WithAsProp<TextAreaProps>, TextAreaState> {
-  static deprecated_className = textAreaClassName;
+export const TextArea: React.FC<WithAsProp<TextAreaProps>> & FluentComponentStaticProps<TextAreaProps> = props => {
+  const context: ProviderContextPrepared = React.useContext(ThemeContext);
+  const { setStart, setEnd } = useTelemetry(TextArea.displayName, context.telemetry);
+
+  setStart();
 
-  static displayName = 'TextArea';
+  const { disabled, accessibility, inverted, resize, fluid, className, design, styles, variables } = props;
 
-  static propTypes = {
-    ...commonPropTypes.createCommon({
-      content: false,
+  const [value, setValue] = useAutoControlled({
+    defaultValue: props.defaultValue,
+    value: props.value,
+    initialValue: props.defaultValue || props.value,
```

Comment:
```suggestion
    initialValue: '',
```

In general, `initialValue` should never take values from other props